### ISSUE TITLE
Why comment out the learning rate scheduler?

### ISSUE THREAD

**Issue by Cowaiyyyy** [chengerr/Model-Guided-Deep-Hyperspectral-Image-Super-resolution#5]
I need to transfer your model to my training framework for training to ensure that the processing methods of our training datasets are consistent. However, when dealing with hyperparameter Settings, I saw that your learning rate scheduler was commented out. Why is that?
